ApplicationInsights/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -428,7 +428,7 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
                 }
 
                 return handled;
-            }
+            }.bind(this);
```

Comment:
is it failing on this._onerror?

Reply:
What does the comment // handled could be typeof function mean?

Reply:
yes it fails at this._onerror. It might be bad to pollute the `this` of onerror. Do you think it's better to call a window instance? if so, will it be ok for multiple instances of sdk initialized?

comment is just clarifying why it is handled !== true instead of !handled

Reply:
Yeah, you can potentially just do window[sdkinstancename]._onerror(). We should save the sdkinstance name somewhere in this class (pass it in through configuration).